src/styles/custom.scss

```diff
@@ -1,18 +1,44 @@
-// Import Bootstrap
-@import '~bootstrap/scss/bootstrap.scss';
-
 // Override Bootstrap Variables
 @import 'variables';
 
+@import '~bootstrap/scss/bootstrap.scss';
+$theme-colors: map-merge($theme-colors, $custom-colors);
+
 // Import Custom Components
 @import './src/styles/components/_brown_fonts.scss';
 @import './src/styles/components/_brown_icons.scss';
 @import './src/styles/components/_brown_footer.scss';
-@import './src/styles/components/_app.scss';
-@import './src/styles/components/_table.scss';
-@import './src/styles/components/_addPub.scss';
+
+.main {
+  min-height: 90vh;
+  padding: 2vw 3vw;
+}
 
 .form-group.required > .form-label:after {
-  content: '*';
+  content: ' *';
```

Comment:
For my own understanding, what does adding a space do here?

Reply:
So if you open up the form modal, all the required fields have " *" next to it if it's a required field. The space in this case is just a space. So instead of `DOI*` we get `DOI *`

<img width="566" alt="Screenshot 2024-03-05 at 4 42 23 PM" src="https://github.com/brown-ccv/ccv-pubs-frontend/assets/5270855/c7d38695-18aa-4ea7-bc9c-781aeb2eb64c">